hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILESlidesPandaBenchmark.py

```diff
@@ -0,0 +1,149 @@
+            filter_mode=self.filter_mode,
+            transforms_dict=self.get_transforms_dict(PandaDataset.IMAGE_COLUMN),
+            crossval_count=self.crossval_count,
+            crossval_index=self.crossval_index,
+            dataloader_kwargs=self.get_dataloader_kwargs(),
+        )
+
+    def create_model(self) -> SlidesDeepMILModule:
+        self.data_module = self.get_data_module()
+        pooling_layer, num_features = self.get_pooling_layer()
+        outputs_handler = self.get_outputs_handler()
+        # Myronenko et al. 2021 uses a cosine LR scheduler which needs to be defined in the PL module
+        # Hence, inherited `PandaSlidesDeepMILModuleBenchmark` from `SlidesDeepMILModule`
+        deepmil_module = PandaSlidesDeepMILModuleBenchmark(encoder=self.get_model_encoder(),
+                                                           label_column=SlideKey.LABEL,
+                                                           n_classes=self.data_module.train_dataset.N_CLASSES,
+                                                           pooling_layer=pooling_layer,
+                                                           num_features=num_features,
+                                                           dropout_rate=self.dropout_rate,
+                                                           class_weights=self.data_module.class_weights,
+                                                           l_rate=self.l_rate,
+                                                           weight_decay=self.weight_decay,
+                                                           adam_betas=self.adam_betas,
+                                                           is_finetune=self.is_finetune,
+                                                           class_names=self.class_names,
+                                                           outputs_handler=outputs_handler,
+                                                           chunk_size=self.encoding_chunk_size,
+                                                           n_epochs=self.max_epochs)
+        outputs_handler.set_slides_dataset(self.get_slides_dataset())
+        return deepmil_module
+
+
+class PandaSlidesDeepMILModuleBenchmark(SlidesDeepMILModule):
+    def __init__(self, n_epochs: int, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        self.save_hyperparameters()
+        self.n_epochs = n_epochs
+
+    def configure_optimizers(self) -> Dict[str, Any]:           # type: ignore
+        optimizer = optim.AdamW(self.parameters(), lr=self.l_rate, weight_decay=self.weight_decay)
+        scheduler = optim.lr_scheduler.CosineAnnealingLR(optimizer=optimizer, T_max=self.n_epochs, eta_min=0)
+        return {"optimizer": optimizer, "lr_scheduler": scheduler}
+
+
+class SlidesPandaImageNetMILBenchmark(DeepSMILESlidesPandaBenchmark):
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(encoder_type=ImageNetEncoder.__name__, **kwargs)
```

Comment:
Since you have now separated `ImageNetEncoder` and `ImageNetEncoder_Resnet50` in BaseMIL `get_encoder`, this should be `ImageNetEncoder_Resnet50`. In the cpath config, I was using the name `ImageNetEncoder` to return resnet50, hence, here we were still using `ImageNetEncoder`